models/product.model.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+const { productList } = require("../data")
+const ProductSchema = new Schema({
+  image: String,
+  name: String,
+  price: Number,
+  stock: String,
+  delivery: String,
+  club: String,
+  type: String,
+  brand: String,
+})
```

Comment:
L7: Your application front-end shows a price as well as discounted price. So `price` attribute here could be modelled as an object with attributes like MRP, discount, net price, etc.

L8: Since `stock` attribute takes a yes/no binary value, I feel Boolean type would be more appropriate.